server/models/tale.py

```diff
@@ -265,7 +265,7 @@ def setAccessList(self, doc, access, save=False, user=None, force=False,
 
         return doc
 
-    def buildImage(self, tale, user, token):
+    def buildImage(self, tale, user, token, force):
```

Comment:
Could you make it a kwarg with a default set to `False`? It don't think it's gonna be used often with `True` on API level.